lib/Api.js

```diff
@@ -279,6 +279,11 @@ class Api {
                     return this.addPodSpecs(plugin, podSpecs, installOptions);
                 }
             })
+            .then(() => {
+                if (plugin != null && plugin.dir && fs.existsSync(path.join(plugin.dir, 'Package.swift'))) {
```

Comment:
Looking at this PR again, and the one thing I don't really love is the "magic" behaviour based on the existence of a Package.swift file in the plugin folder. It feels safer to rely on an explicit declaration in the plugin.xml file.

We've already overloaded the `<framework>` tag to do a billion things, so adding this wouldn't be totally unreasonable. i.e.,
```xml
<framework src="Package.swift" type="spm" />
```

Alternatively, the `lib-file` tag exists on Android but you could maybe stretch the definition to encompass Swift packages.

Using `source-file` is an option, but feels like that goes back to "magic" behaviour based on file names.